src/Compilers/CSharp/Test/Semantic/Semantics/AmbiguousOverrideTests.cs

```diff
@@ -1180,27 +1180,27 @@ public static void Main(string[] args)
                 // CONSIDER: Dev10 reports CS1957, even though the runtime has no trouble distinguishing the potentially
                 // overridden methods.
 
-                // (6,26): error CS0462: The inherited members 'CG<T>.F(T)' and 'CG<T>.F(T)' have the same signature in type 'EG<T>', so they cannot be overridden
-                //     public override void F(T c)
-                Diagnostic(ErrorCode.ERR_AmbigOverride, "F").WithArguments("CG<T>.F(T)", "CG<T>.F(T)", "EG<T>"),
                 // (15,26): error CS0462: The inherited members 'CG<T>.F(T)' and 'CG<T>.F(T)' have the same signature in type 'EGI', so they cannot be overridden
                 //     public override void F(int c)
-                Diagnostic(ErrorCode.ERR_AmbigOverride, "F").WithArguments("CG<T>.F(T)", "CG<T>.F(T)", "EGI"),
+                Diagnostic(ErrorCode.ERR_AmbigOverride, "F").WithArguments("CG<T>.F(T)", "CG<T>.F(T)", "EGI").WithLocation(15, 26),
+                // (6,26): error CS0462: The inherited members 'CG<T>.F(T)' and 'CG<T>.F(T)' have the same signature in type 'EG<T>', so they cannot be overridden
```

Comment:
>// (6,26): error CS0462: The inherited members 'CG<T>.F(T)' and 'CG<T>.F(T)' have the same signature in type 'EG<T>', so they cannot be overridden [](http://example.com/codeflow?start=16&length=146)

Consider placing this error to its original position to make the diff more informative.